test/unit/apps/rotm/controllers/report.js

```diff
@@ -20,6 +23,10 @@ describe('Report Controller', () => {
     controller.should.have.property('locals').that.is.a('function');
   });
 
+  it('has a validateField method', () => {
+    controller.should.have.property('validateField').that.is.a('function');
```

Comment:
no particular reason to rely on chai here to assert that controller has a validateField property, could replace this with..

```
controller.validateField.is.a('function');
```

As per @lennym's mention that chai will not necessarily throw an exception if the chai api is misused.